src/NUnitFramework/tests/Internal/Filters/OrFilterTests.cs

```diff
@@ -76,6 +76,58 @@ public void ExplicitMatchTest()
             Assert.False(filter.IsExplicitMatch(_yetAnotherFixture));
         }
 
+        [TestCase(new bool[] { false, false }, false, StrictIdFilterForTests.EqualValueFunction.IsExplicitMatch)]
+        [TestCase(new bool[] { true, false }, true, StrictIdFilterForTests.EqualValueFunction.IsExplicitMatch)]
+        [TestCase(new bool[] { false, true }, true, StrictIdFilterForTests.EqualValueFunction.IsExplicitMatch)]
+        [TestCase(new bool[] { true, true }, true, StrictIdFilterForTests.EqualValueFunction.IsExplicitMatch)]
+        [TestCase(new bool[] { false, false }, false, StrictIdFilterForTests.EqualValueFunction.Match)]
+        [TestCase(new bool[] { true, false }, true, StrictIdFilterForTests.EqualValueFunction.Match)]
+        [TestCase(new bool[] { false, true }, true, StrictIdFilterForTests.EqualValueFunction.Match)]
+        [TestCase(new bool[] { true, true }, true, StrictIdFilterForTests.EqualValueFunction.Match)]
+        [TestCase(new bool[] { false, false }, false, StrictIdFilterForTests.EqualValueFunction.Pass)]
+        [TestCase(new bool[] { true, false }, true, StrictIdFilterForTests.EqualValueFunction.Pass)]
+        [TestCase(new bool[] { false, true }, true, StrictIdFilterForTests.EqualValueFunction.Pass)]
+        [TestCase(new bool[] { true, true }, true, StrictIdFilterForTests.EqualValueFunction.Pass)]
+        public void CombineTest(IEnumerable<bool> inputBooleans, bool expectedResult,
+            StrictIdFilterForTests.EqualValueFunction expectedCall)
+        {
+            var filters = new List<StrictIdFilterForTests>();
```

Comment:
Would you add a comment in this and other tests like it - or maybe on the StrictIdFilterForTests class that explains what you are tryiing to test. It really takes a bit of mental exercise to figure it out and we would like folks to understand our tests, which are often used as examples.